### PR TITLE
Add tests for PageLayout

### PR DESCRIPTION
**Pull request by colebemis**
Adds Jests test for PageLayout

Part of https://github.com/github/primer/issues/565

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = {
     '<rootDir>/src/utils/test-deprecations.tsx',
     '<rootDir>/src/utils/test-helpers.tsx'
   ],
-  testMatch: ['<rootDir>/(src|codemods)/__tests__/**/*.test.[jt]s?(x)', '!**/*.types.test.[jt]s?(x)'],
+  testMatch: ['<rootDir>/(src|codemods)/**/*.test.[jt]s?(x)', '!**/*.types.test.[jt]s?(x)'],
```
Comment (siddharthkp) [inline jest.config.js:12]: ooooh collocation 👍

Comment (siddharthkp) [reply jest.config.js:12]: We might want to exclude stories from showing up in coverage with [`collectCoverageFrom`](https://jestjs.io/docs/configuration#collectcoveragefrom-array)

image

File: docs/content/drafts/PageLayout.mdx [hunk 2]
```diff
@@ -252,7 +252,7 @@ See [storybook](https://primer.style/react/storybook?path=/story/layout-pagelayo
     noUnnecessaryDeps: true,
     adaptsToThemes: true,
     adaptsToScreenSizes: true,
-    fullTestCoverage: false,
+    fullTestCoverage: true,
```
Comment (siddharthkp) [inline docs/content/drafts/PageLayout.mdx:255]: Solid work here!

Would be nice to get more coverage on the branches here with all the different options

image

**Review by siddharthkp — CHANGES_REQUESTED** [review]
Looking good! This component is progressing fast! 🚀 

Couple of changes requested:
- Jest config update for coverage
- More branch coverage for props

File: docs/content/drafts/PageLayout.mdx (hunk 2 above)
Comment (colebemis) [reply docs/content/drafts/PageLayout.mdx:255]: Adds more tests to cover these branches 👍  There are a few uncovered branches in some internal components (dividers) but I'd have to write some hacky code to cover those cases. Let me know if you comfortable with this level of code coverage

**Review by siddharthkp — APPROVED** [review]
ship it!

**Pull request merged**